src/api/todos.ts

```diff
@@ -0,0 +1,18 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${6725}`);
```

Comment:
Consider using a more descriptive variable name instead of `6725`. It's not clear what this number represents. 🤔

Reply:
Move all 6725 to the const variable and use it. Check other similar cases.